model/clusters_mgmt/v1/metric_queries_resource.model

```diff
@@ -27,4 +27,9 @@ resource MetricQueries {
 	locator SocketTotalByNodeRolesOS {
 		target SocketTotalByNodeRolesOSMetricQuery
 	}
+
+    //Reference to the resource that retrieves the cluster operator status metrics.
```

Comment:
Are you using spaces for indentation here? Make sure to use tabs only. And add an space between the `//` and the first character of the comment.